cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
+set -e
+
+# setting up colors
+BLU='\033[0;34m'
+YLW='\033[0;33m'
+GRN='\033[0;32m'
+RED='\033[0;31m'
+NOC='\033[0m' # No Color
+echo_info(){
```

Comment:
Nice! I guess I have the same question as other people, which is how we can make it reusable across repos.

Reply:
IMO we should eventually develop some sort of integration testing that is external to these stack repos. As long as we are doing it with scripts and Jenkinsfiles in each of the repositories, then there is potential for testing across stacks to be inconsistent. Moreover, as more and more stacks are created, it would be nice if they could just have a `Crosstest` configuration file or something of that nature (which could potentially be generated by something like the crossplane-cli) that allows them to plugin to a test suite in a simple and straightforward fashion. I have lots of ideas around this that would probably be good to reflect in an issue in `crossplaneio/crossplane`. However, in the meantime I would like to have even this minimal integration testing in place for the three existing stacks because I don't like the idea that PR's are merged without being tested.